dscreator/datasets/timeseries/glomma.py

```diff
@@ -0,0 +1,96 @@
+import logging
+from dataclasses import dataclass
+
+import xarray as xr
+from dataclasses import asdict
+
+from dscreator import utils
+from dscreator.cfarray.attributes import VariableAttrs, DatasetAttrsDiscrete
+from dscreator.datasets.base import TimeseriesDatasetBuilder
+
+
+@dataclass
+class BaterodBuilder(TimeseriesDatasetBuilder):
+    def dataset_attributes(self, ds: xr.Dataset) -> DatasetAttrsDiscrete:
+        """Add ACDD attributes to a xarray dataset
+
+        Add attributes following the Attribute Convention for Data Discovery to a dataset, also see https://adc.met.no/node/96.
+        A good keywords viewer is located here https://gcmd.earthdata.nasa.gov/KeywordViewer.
+        For converting CF standard names erddap also contains a good converter for cf standard names.
+        """
+        return DatasetAttrsDiscrete(
+            title="Sensor station at Baterød in Glomma",
+            summary="Sensor measurements collected by a sensor station at Baterød in Glomma.",
```

Comment:
Maybe "Water quality sensor measurements ... at Baterød in river Glomma."

Reply:
Yeah, I was lazy:)